PiwikTracker/Device.swift

```diff
@@ -0,0 +1,132 @@
+import UIKit
+
+internal struct Device {
+    
+    /// A shared instance holding the properties for the current device
+    static let current:Device = Device(platform: currentPlatform(),
+                                       humanReadablePlatformName: currentHumanReadablePlatformName(),
+                                       osVersion: currentOSVersion(),
+                                       screenSize: currentScreenSize(),
+                                       nativeScreenSize: currentNativeScreenSize())
+    /// The platform name of the device i.e. "iPhone1,1" or "iPad3,6"
+    let platform: String
+    
+    /// A human readable version of the platform name i.e. "iPhone 6 Plus" or "iPad Air 2 (WiFi)"
+    /// Or the platform name in case no human readable name was found.
+    let humanReadablePlatformName: String
```

Comment:
The human readable name could be a function that calculates the human readable version from the platform. And it could return an optional that is nil if no human readable version exists.